lti_consumer/lti_1p3/consumer.py

```diff
@@ -488,6 +492,18 @@ def lti_ags(self):
 
         return self.ags
 
+    @property
+    def lti_nrps(self):
```

Comment:
This is unusual to have a property just to raise an exception if the attribute is None.  Does it simplify much code elsewhere?